flash/image/embedding/model.py

```diff
@@ -110,8 +113,17 @@ def __init__(
             learning_rate=learning_rate,
         )
 
+        @dataclass
+        class LambdaInputTransform(InputTransform):
```

Comment:
Let's just import the one we have in `flash.core.data.io.input_transform` instead :smiley:

Reply:
Ah yes, done! 🎉